### PR TITLE
Dls 5142

### PR DESCRIPTION
**Pull request by JakeReid2020**

### CODE REVIEW

File: app/uk/gov/hmrc/helptosavefrontend/controllers/ReminderController.scala [hunk 1]
```diff
@@ -234,7 +255,29 @@ class ReminderController @Inject() (
...
+                Ok(accountClosed(Some(bckLink),account.closureDate.getOrElse(LocalDate.now())))
+              } else {
+                SeeOther(routes.ReminderController.accountOpengetSelectedRendersPage.url)
+              }
+            }
+            })
+    }(loginContinueURL = routes.ReminderController.selectRemindersSubmit.url)
+  }
+
+  def accountOpengetSelectedRendersPage(): Action[AnyContent] =
```
Comment (lutnos) [inline app/uk/gov/hmrc/helptosavefrontend/controllers/ReminderController.scala:280]: Camel case: that should read accountOpenGetSelectedRendersPage

**Review by lutnos — CHANGES_REQUESTED** [review]
Fine, just one minor change, I think

File: app/uk/gov/hmrc/helptosavefrontend/controllers/ReminderController.scala (hunk 1 above)
Comment (JakeReid2020) [reply app/uk/gov/hmrc/helptosavefrontend/controllers/ReminderController.scala:280]: Sweet Ill amend that now!

Comment (JakeReid2020) [reply app/uk/gov/hmrc/helptosavefrontend/controllers/ReminderController.scala]: Mended those changes

**Review by lutnos — APPROVED** [review]

**Pull request merged**